entrypoint.sh

```diff
@@ -3,24 +3,27 @@
 export HOST=$1
 export USER=$2
 echo "$3" > ./commands.txt
-export DOLLAR="$"
 export CLIENT_ID=$4
 export CLIENT_SECRET=$5
+export KEY_TYPE=$8
+export PORT=$9
+export REMOTE_SHELL=${10}
 
 envsubst < "/root/ssh-client.conf" > "/root/.ssh/config" 
-envsubst < "/root/expect.sh" > "/root/expect-final.sh" 
 PUBLIC_KEY="$(echo $6 | base64 --decode)"
 PRIVATE_KEY="$(echo $7 | base64 --decode)"
-sh -c "cat > /root/.ssh/id_rsa << 'EOL'
+
+sh -c "cat > /root/.ssh/${KEY_TYPE} << 'EOL'
 $PRIVATE_KEY
 EOL"
-sh -c "cat > /root/.ssh/id_rsa.pub << 'EOL'
+sh -c "cat > /root/.ssh/${KEY_TYPE}.pub << 'EOL'
 $PUBLIC_KEY
 EOL"
 
-chmod 0600 /root/.ssh/id_rsa
-echo "***  connecting to ssh targrt: ***\n"
+chmod 0600 /root/.ssh/${KEY_TYPE}
+echo "***  connecting to ssh target: ***\n"
 cat /root/.ssh/config
 echo "***  running commands: ***\n"
 cat ./commands.txt
-/root/expect-final.sh
+
+ssh -v -o "StrictHostKeyChecking no" -p $PORT $USER@$HOST "$REMOTE_SHELL" < commands.txt
```

Comment:
particularly since you're already constructing a `commands.txt`, I was able to just redirect that file to `sh` (the default `REMOTE_SHELL`), and run the commands that way.

Reply:
Yeah, you are probably right, about removing (the no longer needed) expect.
I guess, it is remainder from the dev process. 
I'll try this, asap and approve the PR.
The rest of your input, are also helpful features (as you can see, since making it, I didn't bother with it allot).
Thanks so much.